nemo_skills/evaluation/metrics/translation_metrics.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nemo_skills.evaluation.metrics.base import BaseMetrics, as_float
+from collections import Counter, defaultdict
+
+from sacrebleu import corpus_bleu
+# from comet import download_model, load_from_checkpoint
+
+# +
+class TranslationMetrics(BaseMetrics):
```

Comment:
Do you intend to extend this to compute COMET as well? If it's going to be in a different class we should just call this BLEU.